docs/yml.md

```diff
@@ -248,6 +248,14 @@ The default value is json-file.
     log_driver: "syslog"
     log_driver: "none"
 
+Specify logging options with `log_opt` for the logging driver, as with the ``--log-opt`` option for docker run.
```

Comment:
`docker run` or Docker `run`

And `syslog`

Basically the rule is tics around anything that you type on a command line or see in a file system listing.